tony-cli/build.gradle

```diff
@@ -14,6 +14,7 @@ shadowJar {
     exclude(dependency('com.sun.jersey.contribs:'))
     exclude(dependency('org.apache.hadoop:'))
   }
+  zip64 true
```

Comment:
what does this do?

Reply:
It's needed once the number of files in the jar exceeds 65535 entries: https://github.com/johnrengelman/shadow/issues/107

Reply:
I guess adding the Azkaban dependencies in tony-core tipped the fat jar over 65535 entries.